api/HTMLSourceElement.json

```diff
@@ -96,65 +96,6 @@
           }
         }
       },
-      "keySystem": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSourceElement/keySystem",
-          "support": {
-            "chrome": {
-              "version_added": true
-            },
-            "chrome_android": {
-              "version_added": true
-            },
-            "edge": {
-              "version_added": "≤18"
```

Comment:
This seems to also have been wrong, https://developer.mozilla.org/docs/Web/API/MediaKeySystemAccess/keySystem return false in Edge 18.